django/test/runner.py

```diff
@@ -336,12 +336,20 @@ def run(self, test):
         return result
 
 
-def parallel_type(value):
-    """Parse value passed to the --parallel option."""
+def get_num_test_processes(value=None):
```

Comment:
Rather than `get_num_test_processes()`, I wonder if something like `get_max_test_processes()` might be a better name. This is because the number of test processes can wind up being smaller, e.g. if there are fewer `TestCase` classes. (You also assign to `max_parallel` elsewhere, so there is an awareness of this meaning / caveat.)

Reply:
Updated.